browser/extensions/brave_extension_install_prompt.cc

```diff
@@ -20,7 +20,7 @@ BravePrompt::~BravePrompt() {
 base::string16 BravePrompt::GetDialogTitle() const {
   if (!extensions::BraveExtensionProvider::IsVetted(extension())) {
     if (type_ == ExtensionInstallPrompt::INSTALL_PROMPT ||
-        type_ == ExtensionInstallPrompt::INLINE_INSTALL_PROMPT) {
+        type_ == ExtensionInstallPrompt::WEBSTORE_WIDGET_PROMPT) {
```

Comment:
INLINE_INSTALL_PROMPT doesn't exist, searching history log, their refs change to WEBSOTRE_WIDGET_PROMPT.